packages/vue-prism-editor/src/Editor.ts

```diff
@@ -261,7 +262,7 @@ export const PrismEditor = Vue.extend({
       input.selectionStart = record.selectionStart;
       input.selectionEnd = record.selectionEnd;
 
-      this.$emit('input', record.value);
+      this.$emit('update:modelValue', record.value);
```

Comment:
`this.$emit('input' => 'update:modelValue'`

https://v3.vuejs.org/guide/migration/v-model.html#migration-strategy